.gitignore

```diff
@@ -10,6 +10,7 @@
 *.user
 *.userprefs
 .vs/
+.vscode/
```

Comment:
Can you please separate these changes?

I'm not saying we don't want them, but we should not "merge" two unrelated things. Especially in servicing branches.